src/Perl6/Grammar.nqp

```diff
@@ -4096,6 +4099,10 @@ grammar Perl6::Grammar is HLL::Grammar does STD {
         <?before \d+ <?before \. <.?alpha> > <.worry: "Precedence of ^ is looser than method call; please parenthesize"> >?
     }
     token prefix:sym<⚛>   { <sym>  <O(|%symbolic_unary)> }
+    token prefix:sym<//>  {
+        <?{ nqp::getcomp('Raku').language_revision ge 'e' }>
+        <sym> <O(|%symbolic_unary)>
```

Comment:
I think we gonna be having a problem with `/  /`-kind of regexes. The above `quote:sym</null/>` should probably be kinda conditional with `\s*` for pre-6.e and `\s+` otherwise.

Reply:
I see what you mean.

Fixed with https://github.com/rakudo/rakudo/pull/5040/commits/b50a0d086adb38e1061c369dd54d813f324cda4d  Or do you have a better solution?

Reply:
No, that's exactly what I had in mind. Thanks!